pipeline/backend/local/local.go

```diff
@@ -146,6 +146,7 @@ func (e *local) StartStep(ctx context.Context, step *types.Step, taskUUID string
 // execCommands use step.Image as shell and run the commands in it
 func (e *local) execCommands(ctx context.Context, step *types.Step, state *workflowState, env []string) error {
 	// Prepare commands
+	// TODO use `entrypoint` from yaml
```

Comment:
Was at the beginning a bit confused which yaml you refer to 😅 

```suggestion
	// TODO support `entrypoint` from pipeline config
```